requirements.txt

```diff
@@ -1,3 +1,8 @@
 pytest
+psutil
+git+https://github.com/capnproto/pycapnp.git
+ninja
+pyyaml --global-option='--with-libyaml'
 yapf==0.24.0
+git+https://github.com/litghost/rapidyaml.git@fixup_python_packaging#egg=rapidyaml&subdirectory=api/python
```

Comment:
I don't think we should merge it while it is referencing the private fork.

Reply:
I agree, but I'm going to merge this for now, because the upstream PR is blocked on debugging a Windows build issue.